### PR TITLE
Filter quality #3

### LINKED ISSUE #3: Filter: Quality
Some of the data do have a low quality and should not be included. We can filter those data in the `preprocess`.

- [ ] filter all points with insufficient quality
- [ ] filter out invalid scan lines

### PR DESCRIPTION
**Pull request by EmbeddedHenni**
This patch makes `filter_data(data, qa_percent)` set elements of the `data` object to `numpy.nan` if the corresponding quality value is under the level of the `qa_percent` parameter. Later, `write_to_database()` skips all data object elements that are `numpy.nan` and does not copy them to the database.

This fixes #3

### CODE REVIEW

File: emissionsapi/preprocess.py [hunk 1]
```diff
@@ -108,14 +108,35 @@ def read_file(ncfile):
...
-    """Filter data before processing them further.
+def filter_data(data, qa_percent):
+    """Filter data before processing them further
+    All corresponding values in the Scan objects are set to NaN
+    if the quality requirement is not met
 
     :param data: scan object with data
     :type data: emissionsapi.preprocess.Scan
+    :param qa_percent: quality to filter in percent
+    :type qa_percent:
```
Comment (tibroc) [inline emissionsapi/preprocess.py]: type specification missing

**Comment by tibroc** [comment]
There are a couple of flake8 complaints that should be resolved.

**Comment by EmbeddedHenni** [comment]
ich habe keinen plan wie ich mir hier die changes anschauen kann, die @shaardie und @tibroc requested haben? aber aufgrund des textes habe ich mir gedacht was ihr meint und das mit nem commit nachgebessert

**Comment by EmbeddedHenni** [comment]
wer merged den PR? ich oder der Assignee?

**Comment by TylerBird** [comment]
I think we defined, if there is an assigne, it is his job to merge the code!?

**Comment by tibroc** [comment]
You successfully complied with the flake8 requirements (as you can see from the travis ci results). However, you added the `.vs/` folder tot the `.gitignore` but the folder is still in your code, you would have to remove it (e.g. `git rm`) before I can merge it.

**Pull request merged**